packages/sources/src/addSourceWizard/schemaBuilder.js

```diff
@@ -25,13 +25,24 @@ export const getAdditionalStepFields = (fields, stepKey) => fields.filter(field
 
 export const getNoStepsFields = (fields, additionalStepKeys = []) => fields.filter(field => !field.stepKey || additionalStepKeys.includes(field.stepKey));
 
-export const injectAuthFieldsInfo = (fields, type, auth, applicationName) => fields.map((field) => {
+export const removeRequiredValidator = (validate = []) =>
```

Comment:
Why is this here? Is it possible to use the form-level validation and have the login there instead of doing this.

Reply:
Not sure how. I need to disable required validators on password values when editing (aka attaching application to a source which already has the right credentials) because the password is not obtainable by the API.

Reply:
Can you make the field conditional?

Reply:
But user should be able to change if he wants to do it. (HelperText is also added with a description.)

It's the same flow as the adding, just with pre-filled values.